src/pal/inc/pal.h

```diff
@@ -3649,6 +3649,19 @@ LoadLibraryExW(
         IN /*Reserved*/ HANDLE hFile,
         IN DWORD dwFlags);
 
+PALIMPORT
+HMODULE
+PALAPI
+LoadLibraryDirectW(
```

Comment:
There is a convention to have PAL_ prefix for custom (ie without Win32 equivalent) methods exposed by the PAL.